graphql_compiler/tests/test_compiler.py

```diff
@@ -422,7 +422,7 @@ def test_traverse_and_output(self):
         '''
         expected_sql = NotImplementedError
         expected_cypher = '''
-            CYPHER 9
+            CYPHER
```

Comment:
Please make a separate branch where you change the cypher version.

Reply:
#370

Reply:
will be undoing the change here to avoid merge conflict.